relational_operators/HashJoinOperator.hpp

```diff
@@ -141,6 +141,12 @@ class HashJoinOperator : public RelationalOperator {
     return output_relation_.getID();
   }
 
+  void doneFeedingInputBlocks(const relation_id rel_id) override {
```

Comment:
Nice work catching this! Could you add a comment here to let others know why the override is needed. Thanks!